modules/apps/transfer/keeper/migrations_test.go

```diff
@@ -108,3 +107,16 @@ func (suite *KeeperTestSuite) TestMigratorMigrateTraces() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestMigratorMigrateTracesCorruptionDetection() {
+	// IBCDenom() previously would return "customport/channel-0/uatom", but now should return ibc/{hash}
+	corruptedDenomTrace := transfertypes.DenomTrace{
+		BaseDenom: "customport/channel-0/uatom", Path: "",
+	}
```

Comment:
mega nit but could we separate fields onto different lines for readability, e.g.

```go
corruptedDenomTrace := transfertypes.DenomTrace{
	BaseDenom: "customport/channel-0/uatom", 
        Path:      "",
}
```

Reply:
oh definitely! Didn't even notice it was two fields